docs/introduction/using-form-types.md

```diff
@@ -0,0 +1,255 @@
+This option is used as array with two options (`container_class`, `image`) and an be used to wrap your form_row in `<div class="{{ js_container.container_class }}" data-type="{{ js_container.data_type }}">`
+
+### is_plugin_data_group
+Defaults to `false`.
+This option can be set to `true` in empty FormType if you want to add part of form from plugin.
+
+### render_form_row
+Defaults to `true`.
+If you want to add your FormType and don't render it's `form_label` and `form_errors` you can set this option to false.
+This can be sed to create FormTypes that can be used in other FormTypes like the ones that are listed below
+
+## Form Types
+We created some form types which can help you with creating your own form types.
+Here you can find information about what they do and what options they have
+
+### [GroupType](../../packages/framework/src/Form/GroupType.php)
+Group type is used for creating groups of containers. It is not mapped onto any property and it inherits data into group
+so you can work with forms in this group same as you were before. `GroupType` makes sure to render your fields into nicely
+styled `div` wrap.
+
+`GroupType` comes with few options that you can use for even more comfortable work.
+
+#### `label`
+Label is used for displaying heading of section for example in `CustomerFormType` all user data like name, last name or
+email address are all in section with label 'Personal Data'
+
+### [DisplayOnlyCustomerType](../../packages/framework/src/Form/DisplayOnlyCustomerType.php)
+Displays name of a registered customer along with a link to his/her detail.
+If there is no customer set, `unregistered customer` text will be displayed instead.
+
+### [DisplayOnlyType](../../packages/framework/src/Form/DisplayOnlyType.php)
+Sometimes form needs to only display information but does not need to change and persist this data, for this usages
+there is `DisplayOnlyType` which does not map property onto `entity` and let you to display your own data.
+
+### [DisplayOnlyUrlType](../../packages/framework/src/Form/DisplayOnlyUrlType.php)
+For displaying custom URL based on routing system, there can be used `DisplayOnlyUrlType`.
+
+### [LocalizedFullWidthType](../../packages/framework/src/Form/LocalizedFullWidthType.php)
+For displaying localized field in vertical order of full-width label and inputs, there can be used `LocalizedFullWidthType`.
+
+### [OrderItemsType](../../packages/framework/src/Form/OrderItemsType.php)
+Displays editable table of OrderItems from provided Order.
+
+### [WarningMessageType](../../packages/framework/src/Form/WarningMessageType.php)
+Sometimes the form needs to contain some information that is important for viewer, for this usage there is  `WarningMessageType`
+that shows highlighted message with warning icon.
+
+### [LocalizedType](../../packages/framework/src/Form/Locale/LocalizedType.php)
+Compound type that renders one form of given type for each locale.
+Returns array indexed by locale.
+
+#### entry_type
```

Comment:
```suggestion
#### entry_type option
```
what about this way ? if this make sense please do it for other options

Reply:
BV